### ISSUE TITLE
Getting Started guide

### ISSUE THREAD

**Issue by hyandell** [Netflix/osstracker#7]
A getting started guide would be very useful.

Currently I'm at "./gradlew build" -> CASS_PORT not set. Probably fairly early in the install game and paused while I wait on time to get a Cassandra and Elastic Search instance setup.

Acknowledged that the Docker instance would be one way to get that done :)

**Comment by kakawait** [comment]
You should specify your _Cassandra_ host/port and _ES_ host port like following

By your own configuration, here my sample using _Cassandra_ and _ES_ official docker container on `localhost`

```
./gradlew -PCASS_HOST=localhost -PCASS_PORT=9042 -PES_HOST=localhost -PES_PORT=9200 -Pgithub_oauth=XXXXXXX
```

But you should change `XXXXXXX` by our generated key (using _GitHub Personal Settings_ -> _Personal access tokens_)

--- 

Until #6 manual installation is necessary, is not so hard using _Cassandra_/_ES_ container. Don't forget to generate DDL before starting https://github.com/Netflix/osstracker/tree/master/osstracker-ddl

Create _Cassandra_ and _ES_ container

```
docker run --name cassandra -d -p 9042:9042 cassandra
docker run --name elasticsearch -d -p 9200:9200 -p 9300:9300 elasticsearch
```

Connect to _Cassandra_ from cqlsh

```
docker run -it --link cassandra:cassandra --rm cassandra cqlsh cassandra
```

Then run command on https://github.com/Netflix/osstracker/blob/master/osstracker-ddl/osstracker.cql, in my use case I replace

```
CREATE KEYSPACE osstracker WITH replication = {
  'class': 'NetworkTopologyStrategy',
  'us-east': '3'
};
```

by 

```
CREATE KEYSPACE osstracker WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 };
```

Create _ES_ index using instruction here https://github.com/Netflix/osstracker/blob/master/osstracker-ddl/elasticsearch.txt

**Comment by javiereguiluz** [comment]
@kakawait thanks for your detailed comment, but sadly, I can't make it work. In any case, I agree with @hyandell: this project is missing some installation guide. Thanks!

**Comment by kakawait** [comment]
@javiereguiluz what's your new issue ? (I'm not part of Netflix)

**Comment by DuaneOBrien** [comment]
+1 for a Getting Started and Installation guide.

**Comment by aspyker** [comment]
Thanks for pulling this together folks.  I planned to write the getting started guide, but didn't get there yet.  @kakawait, you got it spot on.

**Comment by kakawait** [comment]
I will try to create _Gist_ or _Asciinema_ from my previous successful installation asap. Feel free to use it after to write a _Getting started_ document. But I really think best solution is to provides full stack container (see #6) :)

**Comment by aspyker** [comment]
@kakawait Can you stick to markdown if possible.  I'd like for you to submit any documentation you created as a PR against the documentation branch I just created.  I'd like to avoid using a Wiki.  An Asciicema would be great as well.

Place we can start to add to:
https://github.com/Netflix/osstracker/blob/documentation/osstracker-docs/README.md

**Comment by kakawait** [comment]
@aspyker Done feel free to polish it.
@javiereguiluz documentation inside PR is based on new fresh installation! So tested (*Mac* + *docker-machine*) and works perfectly. But be careful I teak some things (`build.gradle`, *DDL*) so does not work with `master` branch

**Comment by javiereguiluz** [comment]
@kakawait thanks for the guide. I'll try to install this application again following your guide. Thanks!

**Comment by kakawait** [comment]
![screen shot 2016-02-14 at 13 07 04](https://cloud.githubusercontent.com/assets/275609/13033570/e8667754-d31b-11e5-8b51-a9340c564d15.png)
